docker/docker-compose.suite-ci.yml

```diff
@@ -1,11 +1,11 @@
-version: "3"
+version: '3'
```

Comment:
why the change to single quotes? Is there any particular reason or only preferences?

Reply:
I guess that is related to prettier reformat (`docker` folder has been removed from `.prettierignore` in this PR)

Reply:
yea, I guess it was in ignore because prettier was broken for `yml` files (bad spacing size), now it is fixed so we should use it because of consistency

Reply:
overrided prettier config for `yml` files to use double quotes, which are more standart for `yml` I belive